src/components/Icon/Icon.tsx

```diff
@@ -50,7 +50,7 @@ export type IconProps = {
 }
 
 /**
- * UI component for displaying different icon packs (Ant, Feather, Phosphor) and custom SVGs
+ * UI component to display different icon packs (Ant, Feather, Phosphor) and custom SVGs
```

Comment:
In this sentence, the absence of "for" makes it more concise and suggests that the UI component is capable of displaying different icon packs and custom SVGs, but it doesn't necessarily imply that it is designed solely for that purpose. It's a statement of functionality without specifying the primary purpose.

In practical terms, both sentences convey a similar idea, but the first one emphasizes the specific purpose or intent of the UI component, while the second one simply states what it can do. The choice between them depends on the context and how you want to frame the information.

Reply:
Also, this comment is a little bit out of scope. I would try to keep modifications to other components only if strictly necessary.